### DISCUSSION TITLE
How do I make more than two text labels per cell?

### DISCUSSION THREAD

**Discussion by sendhelpp** [sst-inc/help#24] [Q&A - iOS] [answered]
I can only configure two text labels per cell, when I have 3 variables so I'm not sure what to do. Please teach me how to add more labels per cell or a work around.

**Comment by theboi** [comment]
@granwyntan

**Comment by granwyntan** [comment, marked as answer]
Great that you would like to extend your learning.

1. Ensure you have more properties within your struct which you could display within your TableViewController.
<img width="596" alt="Screenshot 2021-08-08 at 13 15 02" src="https://user-images.githubusercontent.com/61010627/128621640-7f20e901-d23c-47a8-bdd7-e730652a210f.png">
2. Select Custom instead of Subtitle for your cell type, in attributes inspector
<img width="221" alt="Screenshot 2021-08-08 at 13 14 01" src="https://user-images.githubusercontent.com/61010627/128621619-5a6c9d92-7deb-4c1f-8ef1-437e56656b56.png">
3. Command+N and Create a UITableViewCell file
<img width="744" alt="Screenshot 2021-08-08 at 13 15 42" src="https://user-images.githubusercontent.com/61010627/128621655-b477fe5e-cc88-43e2-a9a6-a3399c85be12.png">
<img width="762" alt="Screenshot 2021-08-08 at 13 15 58" src="https://user-images.githubusercontent.com/61010627/128621660-c8380a7b-0a47-4202-a3eb-2105a53bfedc.png">
4. Change the TableViewCell class via identity inspector
<img width="900" alt="Screenshot 2021-08-08 at 13 39 38" src="https://user-images.githubusercontent.com/61010627/128622063-5b542c7b-6780-462e-8c4f-abc2102d036a.png">
5. Double click on the cell and open TableViewCell.swift in the assistant editor
<img width="910" alt="Screenshot 2021-08-08 at 13 40 44" src="https://user-images.githubusercontent.com/61010627/128622089-fec9323f-c1bf-45f7-9e6a-75c7d3d4a1e6.png">
6. Add labels in and connect them via outlets
<img width="907" alt="Screenshot 2021-08-08 at 13 44 09" src="https://user-images.githubusercontent.com/61010627/128622136-fd06200a-3a41-48e8-a971-8d6bc157ed7f.png">
7. Set as custom cell and change the text for labels in the cellForRowAt function
<img width="361" alt="Screenshot 2021-08-08 at 13 48 22" src="https://user-images.githubusercontent.com/61010627/128622200-a1127071-38ae-41b9-aeaa-ef48894fe4b2.png">
8. You might have to change the alerts, and other stuff, they'll prompt u with errors and u can fix them
<br>
You're good to go.

**Reply by sendhelpp** [reply]
[ok thanks
